motoko/simple_to_do/src/simple_to_do/public/index.js

```diff
@@ -0,0 +1,5 @@
+import simple_to_do from 'ic:canisters/simple_to_do';
+
+simple_to_do.addTodo(window.prompt("Enter your task")).then(Todos => {
+  window.alert(Todos);
+});
```

Comment:
I'd like to revisit this example at some point to use an off-the-shelf front-end from [TodoMVC](http://todomvc.com).

Reply:
Yeah, something that works would be cool. I tried to fake something up to replace the greet/greeting frontend piece but didn't want to add a full React type thing so I punted.